libs/core/src/lib/dynamic-side-content/dynamic-side-content.component.html

```diff
@@ -0,0 +1,11 @@
+<ng-container *ngIf="_isSideBefore" [ngTemplateOutlet]="sideTmplRef"></ng-container>
```

Comment:
Lets use full names, maybe just `sideTemplate`?

Reply:
renamed it